### PR TITLE
Completed Extended File Support for Flashcard Generator

### PR DESCRIPTION
**Pull request by youngsunlee07**
Implemented support for multiple file attachments in the Flashcard Generator as part of Epic 7.2. This update enhances the application's ability to handle various file types, improving flexibility and user experience. Requesting code review.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,10 @@ COPY requirements.txt /code/requirements.txt
 
 RUN pip install --no-cache-dir -r /code/requirements.txt
 
-COPY ./app /code/app
+COPY ./app /code/app 
+
+# .env 파일 복사
```
Comment (hash2004) [inline Dockerfile:12]: unnecessary comments should be removed.

File: app/main.py [hunk 2]
```diff
@@ -10,7 +10,12 @@
 import os
 from dotenv import load_dotenv, find_dotenv
 
-load_dotenv(find_dotenv())
+load_dotenv(find_dotenv()) 
+
+# 추가된 부분
```
Comment (hash2004) [inline app/main.py:15]: remove unnecessary comments

File: requirements.txt
Comment (hash2004) [inline requirements.txt:1]: include fastapi[standard]

**Review by hash2004 — CHANGES_REQUESTED** [review]
Awesome awesome work! Solve the minor issues and submit your PR with proper screenshots (according to the PR template) and tests. Rest looks clean!

**Comment by youngsunlee07** [comment]
I have incorporated the feedback and pushed the changes. Please review the updated code.

**Comment by AaronSosaRamos** [comment]
Closed as this feature is not required anymore.

**Pull request closed without merging**